addons/mail/data/ir_attachment_data.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="mail.avatar_placeholder" model="ir.attachment">
+            <field name="public" eval="True"/>
+            <field name="name">avatar_placeholder.jpg</field>
+            <field name="res_model">ir.ui.view</field>
+            <field name="type">url</field>
+            <field name="url">/base/static/img/avatar_grey.png</field>
+        </record>
+    </data>
+</odoo>
```

Comment:
This should make use of the following instead

```
    def _avatar_get_placeholder_path(self):
        return "base/static/img/avatar_grey.png"
```

Reply:
Moreover it is going to break upgrade, and having a master data just for that is unnecessary.